deltacat/utils/ray_utils/concurrency.py

```diff
@@ -107,4 +107,11 @@ def round_robin_options_provider(
     assert resource_keys, f"No resource keys given to round robin!"
     resource_key_index = i % len(resource_keys)
     key = resource_keys[resource_key_index]
-    return {"resources": {key: resource_amount_provider(resource_key_index)}}
+    if kwargs.get("pg_config"):
+        #use pg and bundle id for fault-tolerant round-robin
+        opt = {"resources": {key: resource_amount_provider(resource_key_index)}}
```

Comment:
can this line be omitted or providing bundle index in `resources` behave same way as specifying a new scheduling strategy object?

Reply:
same as previous comment